linodecli/arg_helpers.py

```diff
@@ -338,6 +325,18 @@ def action_help(cli, command, action):
             print("You may filter results with:")
             for attr in filterable_attrs:
                 print(f"  --{attr.name}")
+        return
+    if op.args:
```

Comment:
```suggestion
    elif op.args:
```

Might be easier to understand this way?